mkosi/__init__.py

```diff
@@ -5577,7 +5738,7 @@ def print_summary(args: CommandLineArguments) -> None:
 
     MkosiPrinter.info("                 Read-only: " + yes_no(args.read_only))
     detail = " ({})".format(args.compress) if args.compress and not isinstance(args.compress, bool) else ""
-    MkosiPrinter.info("            FS Compression: " + yes_no(args.compress) + detail)
+    MkosiPrinter.info("            FS Compression: " + yes_no(cast(bool, args.compress)) + detail)
```

Comment:
Why the cast here, if the signature is changed above to `Optional[bool]`?

Reply:
Not sure what you mean, `compress` was changed to `Union[str, bool, None]`.

Reply:
Sorry, my bad. Guess I wasn't fully awake yet, when I looked over this.